packages/expo-updates/ios/UpdatesRuntimeOverrides.swift

```diff
@@ -0,0 +1,54 @@
+// Copyright 2015-present 650 Industries. All rights reserved.
+
+import ExpoModulesCore
+
+/**
+ Runtime overridable config from build time UpdatesConfig
+ */
+public struct UpdatesRuntimeOverrides: Record, Codable {
```

Comment:
optional: consider separate internal representation (what's stored in defaults) from external (what's passed into the JS method) so that migrations are clearer (breaking changes less likely). can do fancy kotlin/swift extension methods to convert.